ClrMD.Extensions/ClrDynamic.cs

```diff
@@ -91,12 +94,14 @@ public object SimpleDisplayValue
             get
             {
                 if (Type.IsEnum)
-                    return Type.GetEnumName(SimpleValue) ?? SimpleValue.ToString();
+                {
+                    return Type.AsEnum().EnumerateValues().FirstOrDefault(f => Convert.ToInt64(f.Value) == Convert.ToInt64(SimpleValue)).Name ?? SimpleValue.ToString();
```

Comment:
This probably won't be a bottleneck, but `Type.AsEnum().EnumerateValues().FirstOrDefault` seems inefficient. That said I don't know how `Type.GetEnumName` used to work, maybe it was already implemented with something simliar.